video/out/gpu/libmpv_gpu.c

```diff
@@ -32,14 +32,18 @@ static const struct native_resource_entry native_resource_map[] = {
         .name = "wl",
         .size = 0,
     },
-    [MPV_RENDER_PARAM_DRM_DISPLAY] = {
+    [MPV_RENDER_PARAM_DRM_DISPLAY] = { // Deprecated
         .name = "drm_params",
         .size = sizeof (mpv_opengl_drm_params),
     },
```

Comment:
I forgot, are unknown parameters ignored, or do they cause an error? If the former, remove this.

Reply:
OK, I think it's just ignored if unknown. So this should be removed, it serves no purpose anymore, but will cause the generic code to perform memory accesses. (Which may be out of bounds with old clients compiled against the pre-ABI break struct.)

Reply:
removed